src/main/java/website/frontrow/Launcher.java

```diff
@@ -62,7 +62,7 @@ public static void main(String[] args) throws IOException
      * @param filename The file name of the level to load.
      * @param playerCount The amount of players in this game.
      */
-    @SuppressWarnings("methodlength") // We need to make a GameFactory and UIBuilder
+    @SuppressWarnings("methodlength")
```

Comment:
I made it shorter by creating a new method, keeping it organized and easy to read.